### ISSUE TITLE
HtmlHelperOptions.ClientValidationEnabled

### ISSUE THREAD

**Issue by Ponant** [aspnet/Docs#10957]
This is very helpful in my apps, so I think it should be useful for others in the docs. It is not working in Razor Pages though yet 
```
services.AddMvc().AddViewOptions(o =>
            {
                if (_env.IsDevelopment())
                {
                    o.HtmlHelperOptions.ClientValidationEnabled = false;
                }
            })
```

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 0156ffc3-47ac-d5da-fbc2-9ff5018b071b
* Version Independent ID: 615db388-a244-41f7-21b5-a4085d9ed479
* Content: [Model validation in ASP.NET Core MVC](https://docs.microsoft.com/en-us/aspnet/core/mvc/models/validation?view=aspnetcore-2.2#client-side-validation)
* Content Source: [aspnetcore/mvc/models/validation.md](https://github.com/aspnet/Docs/blob/master/aspnetcore/mvc/models/validation.md)
* Product: **aspnet-core**
* GitHub Login: @tdykstra
* Microsoft Alias: **riande**

**Comment by Ponant** [comment]
j

**Comment by Rick-Anderson** [comment]
@Ponant 	that's a great idea. That's much better than disabling JavaScript on all your browsers - only to have to reset them later.

@rynowak 	should I open a product bug for `HtmlHelperOptions.ClientValidationEnabled = false;` not working in Razor Pages?

**Comment by Ponant** [comment]
@Rick-Anderson you are welcome. I already opened the issuebut no answer yet if this is a bug or this extension is only valid by construction to traditional MVC.
https://github.com/aspnet/AspNetCore/issues/7576
In any case, yes it is a super useful feature to have and having it absent from razor pages would be a pity. At the moment I came back to disabling js on the browser to have this effectively working.
